v0/destinations/facebook_pixel/transform.js

```diff
@@ -493,8 +492,12 @@ const responseBuilderSimple = (
           break;
         case "products searched":
           const query = message.properties?.query;
-          if (query && typeof query !== "string" && typeof query !== "number") {
-            throw new CustomError("'query' should be in (string (or) number) format only", 400);
+          const validQueryType = ["string", "number", "boolean"];
```

Comment:
can you please add the reference url for valid query types in facebook pixel?

Reply:
But, according to Facebook Pixel [here](https://developers.facebook.com/docs/meta-pixel/reference#:~:text=an%20exact%20value.-,search_string,-String), they stated it must be in `string` format. But as they are accepting `number` and `boolean` also. So, we included them too.

Reply:
Added reference URL